DESCRIPTION

```diff
@@ -116,7 +115,7 @@ Suggests:
 Remotes:
     r-lib/rlang,
     rstudio/bslib,
-    rstudio/htmltools
+    rstudio/htmltools#212
```

Comment:
Needed for the `tagGetAttribute()` usage, which isn't necessary, but it'd be nice to move in that direction